src/components/DeleteModal/DeleteModal.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import Button from '@mui/material/Button';
+import ClearIcon from '@mui/icons-material/Clear';
+import CheckIcon from '@mui/icons-material/Check';
+
+const DeleteModal = ({showDeleteModal,setShowDeleteModal,client,handleDelete}) => (
+    <Dialog
+    open={showDeleteModal}
+    aria-labelledby="dialog-title"
+    aria-describedby="dialog-description"
+    onClose={() => setShowDeleteModal(false)}
+  >
+   
+   <DialogTitle id="dialog-tile">Delete Client?</DialogTitle>
+   
+   <DialogContent>
+      <DialogContentText id="dialog-description">
+      {`You're about to delete ${client.person} from your client list, do you wish to continue?`}
+      </DialogContentText>
+    </DialogContent>
+   
+    <DialogActions>
+      <Button
+        variant="outlined"
+        color="error"
+        endIcon={<ClearIcon />}
+        onClick={() => setShowDeleteModal(false)}
+      >
+        NO
+      </Button>
+
+        <Button
```

Comment:
Just a small formatting issue, this 2nd button should be left one more tab so it lines up with the 1st button code.

We do have a Github Actions check setup for formatting/linting.  Most likely you will need to run the command `npm run prettier:run` in your terminal to format this code correctly to pass the Github Actions testing.